scripts/ec/benchmark/solr.rb

```diff
@@ -0,0 +1,12 @@
+require 'yaml'
+created_objects = YAML.load_file("testdata/created-objects.yml")
+
+org = Chef::Config['chef_server_url'].split("/")[-1]
+nodes = created_objects["orgs"][org]["nodes"]
+
+nodes.each do |node|
```

Comment:
I think we will probably want to test along to axis of variation:

- The number of concurrent writers
- The size of the objects being written